### PR TITLE
Add more details for COSI for AWS

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 1]
```diff
@@ -219,6 +214,26 @@ For example, `--restore-until=123`.
 CREATE DATABASE mydatabase;
 ----
 
+[[restore-backup-aws-s3]]
+==== Restore a backup located in an AWS S3 storage
```
Comment (docrozza) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:218]: Not picked up as a link on the left side of the page - should it be === instead

File: modules/ROOT/pages/backup-restore/offline-backup.adoc [hunk 2]
```diff
@@ -96,11 +96,14 @@ The `neo4j-admin database dump` command has the following options:
 [NOTE]
 ====
 As of Neo4j 5.19, the `--to-path=<path>` option can also dump databases to AWS S3 URIs.
-Neo4j uses link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[default AWS credentials] to access AWS S3 URIs. 
+Neo4j uses link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[default AWS credentials] to access AWS S3 URIs.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/backup-restore/offline-backup.adoc:99]: I was actually thinking of removing this 1.a step.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/tools/neo4j-admin/consistency-checker.adoc [hunk 3]
```diff
@@ -125,8 +125,8 @@ Value can be plain numbers, like `10000000` or e.g. `20G` for 20 gigabytes, or e
 [NOTE]
 ====
 As of Neo4j 5.19, the `--from-path=<path>` option can also check artifacts in AWS S3 URIs.
-Neo4j uses link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[default AWS credentials] to access AWS S3 URIs. 
-==== 
+For more information, see <<
```
Comment (docrozza) [inline modules/ROOT/pages/tools/neo4j-admin/consistency-checker.adoc:128]: this looks like it's missing something

File: modules/ROOT/partials/aws-s3-credentials.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
+. Ensure that the AWS CLI is installed and configured with the necessary credentials.
+.. Install the AWS CLI by following the instructions in the AWS official documentation -- link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[Install the AWS CLI version 2].
+.. Use `aws configure` command to set your `aws_access_key_id` and `aws_secret_access_key` from AWS.
+.. Create an S3 bucket and a directory to store the backup files using the AWS CLI:
++
+[source,shell]
+----
+aws s3 mb --region=us-east-1 s3://myBucket
+aws s3api put-object --bucket mat --key myDirectory/
```
Comment (docrozza) [inline modules/ROOT/partials/aws-s3-credentials.adoc:9]: `--bucket myBucket`

Comment (NataliaIvakina) [inline modules/ROOT/partials/aws-s3-credentials.adoc:9]:
```suggestion
aws s3api put-object --bucket myBucket mat --key myDirectory/
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/tools/neo4j-admin/consistency-checker.adoc (hunk 3 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/consistency-checker.adoc:128]:
```suggestion
For more information, see <<check-database-from-aws-s3-uris>>.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/partials/aws-s3-credentials.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
+. Ensure that the AWS CLI is installed and configured with the necessary credentials.
+.. Install the AWS CLI by following the instructions in the AWS official documentation -- link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[Install the AWS CLI version 2].
+.. Use `aws configure` command to set your `aws_access_key_id` and `aws_secret_access_key` from AWS.
+.. Create an S3 bucket and a directory to store the backup files using the AWS CLI:
++
+[source,shell]
+----
+aws s3 mb --region=us-east-1 s3://myBucket
+aws s3api put-object --bucket myBucket mat --key myDirectory/
```
Comment (docrozza) [inline modules/ROOT/partials/aws-s3-credentials.adoc:9]: Sorry - meant that it should be:
`aws s3api put-object --bucket myBucket --key myDirectory/`

i.e. no mat

Comment (NataliaIvakina) [inline modules/ROOT/partials/aws-s3-credentials.adoc:9]:
```suggestion
aws s3api put-object --bucket myBucket --key myDirectory/
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1574.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1574.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Review by docrozza — APPROVED** [review]

**Pull request merged**